src/views/Package/components/PackageDocs/PackageDocs.tsx

```diff
@@ -20,12 +20,15 @@ export const appendItem = (itemTree: Item[], item: Element): Item[] => {
 
   const { headingId, headingTitle = "", headingLevel = "100" } = item.dataset;
   const level = parseInt(headingLevel);
-  if (level > 3) {
+
+  // Don't create nav items for items with no title / url
+  if (level > 3 || !headingTitle || !headingId) {
     return itemTree;
   }
 
   const last = itemTree[itemTree.length - 1];
-  if (typeof last === "undefined" || last.level === level) {
+
+  if (last == null || last.level >= level) {
```

Comment:
This change specifically prevents a heading from being a descendant of a lower heading i.e `h1` should not be the child of an `h2` on the nav tree